flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/plan/util/ExplodeFunctionUtil.scala

```diff
@@ -54,6 +55,15 @@ abstract class ExplodeTableFunction[T] extends TableFunction[T] {
   }
 }
 
+class MapExplodeTableFunc extends TableFunction[Object]{
```

Comment:
space before brace

Reply:
I'm not sure making this table function returning object and let framework rely on information somewhere else to perform right is good choice. This will make the codes fragile.

Reply:
> space before brace

fixed

> I'm not sure making this table function returning object and let framework rely on information somewhere else to perform right is good choice. This will make the codes fragile.

The UNNEST function should be able to explode any type of key and value types. This is the reason why I'm using Object type here. Basically, I use the same approach which used for Array types (for the arrays used the ObjectExplodeTableFunc)

Reply:
`ObjectExplodeTableFunc` is not directly used by all arrays which want to be unnested. It's only used by the array which has object array element type, .e.g. Array(object[]).

Reply:
Thanks for your comments, I've fixed the TableFunction and now it returns Row